examples/ads.amp.html

```diff
@@ -1090,6 +1091,13 @@ <h2>eADV</h2>
       data-u="l0o0c0a8l%3Ahtos">
   </amp-ad>
 
+  <h2>Empower</h2>
```

Comment:
I noticed this example is not rendering a creative. Would it be possible to have a test creative show here? It can help us with future debugging.

Reply:
Done! 👍